src/translations/en.json

```diff
@@ -1246,7 +1246,7 @@
             "learn": "Learn more",
             "ignore": "Ignore",
             "unignore": "Unignore",
-            "breaks_in_version": "This will break in version {version}. Fix this issue before upgrading.",
+            "breaks_in_version": "This will no longer work in version {version}. Please fix this issue before upgrading.",
```

Comment:
I had removed please so it would fit in one line

Reply:
with `no longer work` being longer than `break`. I don think it will be one line 

Also the width is dependent on the title of the issue

Reply:
Shorter:

`This stops working in version {version}. Please address before upgrading.`

Reply:
```suggestion
            "breaks_in_version": "This stops working in version {version}. Please address before upgrading.",
```